storops/unity/resource/__init__.py

```diff
@@ -112,8 +112,9 @@ def _is_updated(self):
                     len(self.get_preloaded_prop_keys()) > 1)
             else:
                 # Return False when only id is parsed
-                ret = not (len(self._parsed_resource) == 1 and
-                           len(self.property_names()) > 1)
+                ret = not (self.parsed_resource is None or

```

Comment:
So, in case of the parsed_resource is None,  the ret should be False.  Please move this logic to line 101.  Otherwise, there will be exception thrown in lone 106.